datalad_neuroimaging/extractors/fslfeat.py

```diff
@@ -0,0 +1,276 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""FSL Feat analysis configuration and result metadata extractor"""
+from __future__ import absolute_import
+
+from os.path import join as opj
+from os.path import exists
+from os.path import basename
+from os.path import dirname
+from os.path import relpath
```

Comment:
not for a change here, just sharing that I started to like/use more a convention `import os.path as op` and then accessing them all from `op.` -- makes it still short and clear, while later we might even come up with our "extended" `datalad.support.path` and change all the imports in one shot